app/models/manageiq/providers/autosde/storage_manager/volume_mapping.rb

```diff
@@ -1,6 +1,16 @@
 class ManageIQ::Providers::Autosde::StorageManager::VolumeMapping < ::VolumeMapping
   supports :create
 
+  supports :delete do
+    unsupported_reason_add(:delete, _("The Volume mapping is not connected to an active Manager")) if ext_management_system.nil?
+  end
+
+  def raw_delete_volume_mapping
+    ems = ext_management_system
+    ems.autosde_client.StorageHostVolumeMappingApi.storage_hosts_mapping_pk_delete(ems_ref)
```

Comment:
Really minor but in the `raw_create_*` method below you do `ext_management_system.autosde_client...`, best we should stay consistent

Reply:
@agrare done